src/drivers/dshot/DShot.h

```diff
@@ -52,7 +52,6 @@ using namespace time_literals;
 static constexpr unsigned int DSHOT150  =  150000u;
 static constexpr unsigned int DSHOT300  =  300000u;
 static constexpr unsigned int DSHOT600  =  600000u;
-static constexpr unsigned int DSHOT1200 = 1200000u;
```

Comment:
One idea (if anyone cares) is we could keep support in the actual driver, but not expose it via parameter metadata.